src/backend/common/helpers/insights_helper.py

```diff
@@ -242,16 +243,25 @@ def _generateMatchData(self, match: Match, event: Event) -> Dict:
         }
 
     @classmethod
-    def _sortTeamWinsDict(self, wins_dict):
+    def _sortTeamWinsDict(
+        self, wins_dict: Dict[TeamKey, int]
+    ) -> List[Tuple[int, List[TeamKey]]]:
```

Comment:
I think it's actually

```suggestion
    ) -> List[Tuple[int, List[int]]]:
```

Reply:
And update the comment below

Reply:
The raw data is

```
[[46, ["frc2713"]], [36, ["frc88"]], [33, ["frc230", "frc3467"]], [32, ["frc125"]], [31, ["frc1757", "frc1768", "frc2067", "frc8013", "frc9055"]], ...
```